backend/src/main/java/com/woowacourse/ternoko/service/ReservationService.java

```diff
@@ -47,6 +50,11 @@ public Long create(final Long coachId, final ReservationRequest reservationReque
             formItem.addInterview(savedInterview);
         }
 
+        AvailableDateTime availableDateTime = availableDateTimeRepository.findByCoachIdAndInterviewDateTime(
+                        coachId, reservationRequest.getInterviewDatetime())
+                .orElseThrow(() -> new InvalidReservationDateException(ExceptionType.INVALID_AVAILABLE_DATE_TIME));
```

Comment:
static import하면 더 깔끔해질 것 같습니다!
```suggestion
                .orElseThrow(() -> new InvalidReservationDateException(INVALID_AVAILABLE_DATE_TIME));
```

Reply:
static import로 수정해보았습니다.